utils/release/lock-master.mjs

```diff
@@ -4,32 +4,36 @@ const REPO_OWNER = 'coveo';
 const REPO_NAME = 'cli';
 const MAIN_BRANCH_NAME = 'master';
 
-export const limitWriteAccessToBot = () =>
-  changeBranchRestrictions({
-    users: [],
-    apps: ['developer-experience-bot'],
-    teams: [],
-  });
+export const limitWriteAccessToBot = () => changeBranchRestrictions(true);
 
 export const removeWriteAccessRestrictions = () =>
-  changeBranchRestrictions(null);
+  changeBranchRestrictions(false);
 
-async function changeBranchRestrictions(restrictions) {
+async function changeBranchRestrictions(onlyBot) {
```

Comment:
Not sure if its documented elsewhere but a brief doc on why this function is necessary and how it helps would be useful

Reply:
Yeah, the whole release process is in dire need of documentation, but no time has been allocated to that.
I'll write a small comment near the function just as a reminder tho.

Reply:
https://github.com/coveo/cli/pull/1251/files#diff-521eca7410150ac5cafa6afc7800332f5351d26b89e3b9f4245fd3c3d1eeb146R1-R5